packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -390,59 +427,75 @@ function toggleAllContainerGroups(value: boolean) {
               </td>
             </tr>
           {/if}
+
           <!-- Display each container of this group -->
           {#if containerGroup.expanded}
             {#each containerGroup.containers as container, index}
+              <!-- Show the type if it's standalone or not -->
               <tr class="group h-12 bg-zinc-900 hover:bg-zinc-700">
                 <td
                   class="{containerGroup.type === ContainerGroupInfoTypeUI.STANDALONE ? 'rounded-tl-lg' : ''} {index ===
                   containerGroup.containers.length - 1
                     ? 'rounded-bl-lg'
                     : ''}">
                 </td>
+
+                <!-- Checkbox -->
                 <td class="px-2">
                   <input
                     type="checkbox"
                     bind:checked="{container.selected}"
-                    class="cursor-pointer invert hue-rotate-[218deg] brightness-75" />
+                    class=" cursor-pointer invert hue-rotate-[218deg] bg-white-100 rounded-sm" />
                 </td>
-                <td class="flex flex-row justify-center h-12">
+
+                <!-- Container icon + state -->
+                <td class="px-2">
                   <ContainerIcon state="{container.state}" />
                 </td>
-                <td class="whitespace-nowrap hover:cursor-pointer" on:click="{() => openDetailsContainer(container)}">
-                  <div class="flex items-center">
-                    <div class="">
-                      <div class="flex flex-nowrap">
-                        <div class="text-sm text-gray-200 overflow-hidden text-ellipsis" title="{container.name}">
-                          {container.name}
-                        </div>
-                        <div
-                          class="pl-2 text-sm text-violet-400 overflow-hidden text-ellipsis"
-                          title="{container.image}">
-                          {container.image}
-                        </div>
+
+                <!-- The entire <td> is clickable -->
+                <td
+                  class="p-2 whitespace-nowrap hover:cursor-pointer"
+                  on:click="{() => openDetailsContainer(container)}">
+                  <div class="">
+                    <!-- Container name -->
+                    <div class="flex flex-row">
+                      <div class="text-sm text-gray-200 overflow-hidden text-ellipsis" title="{container.name}">
+                        {container.name}
                       </div>
-                      <div class="flex flex-row text-xs font-extra-light text-gray-500">
-                        <div>{container.state}</div>
-                        <!-- Hide in case of single engines-->
-                        {#if multipleEngines}
```

Comment:
from what I see, the engine is always displayed while before it was displayed only if we had in the list more than one engine
so I would stick to not displaying it as it's most of the time useless

Reply:
Good idea, should we hide the column completely if there are no multiple engines? @mairin what do you think